<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectAcc</class>
 <widget class="QDialog" name="SelectAcc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/img/mainbg.png)</string>
  </property>
  <widget class="QPushButton" name="confirmAdd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AccountNoInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="AccountNoBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Account No.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmountBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Amount&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AmountInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="OwnerBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Owner&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OwnerInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="SelectBankCombo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Krungthep Bank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Krungsri Bank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Krungthai Bank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Siam Commercial Bank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Government Saving Bank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thanachart Bank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TMB Bank</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kasikorn Bank</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="BankBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Bank&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorMsg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>275</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#ff0000;&quot;&gt;Error : Wrong Input&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
